pkg/rbac/tr.json

```diff
@@ -0,0 +1,43 @@
+{
+    "kind": "TokenReview",
+    "apiVersion": "authentication.k8s.io/v1",
+    "metadata": {
+        "creationTimestamp": null,
+        "managedFields": [
+            {
+                "manager": "main",
+                "operation": "Update",
+                "apiVersion": "authentication.k8s.io/v1",
+                "time": "2022-06-21T16:39:09Z",
+                "fieldsType": "FieldsV1",
+                "fieldsV1": {
+                    "f:spec": {
+                        "f:token": {}
+                    }
+                }
+            }
+        ]
+    },
+    "spec": {
+        "token": "sha256~oKUU_nxSIAUjBJHWe7cCz6iLQ2jDn-WZ2d--S4aMda8"
```

Comment:
To avoid getting flagged by secure engineering scans, change this to a string that makes it clear that this is a mock token.